build.gradle

```
@@ -218,7 +219,7 @@ subprojects {
           // effectively uses whatever slf4j-api version is provided by IntelliJ.
           // SLF4J does NOT guarantee, however, that slf4j-api version X will work with any slf4j implementation version Y for X != Y.
           // To avoid a clash between JGit's slf4j-api and Intellij's slf4j implementation, we need to exclude the former.
```

Just to refine my comment while you're here:
```suggestion
          // To avoid a clash between JGit's slf4j-api and Intellij's slf4j implementation (and also between JGit's slf4j-api and Intellij's slf4j-api), we need to exclude the former.
```